ownCloud Intents/CreateFolderIntentHandler.swift

```diff
@@ -22,7 +22,36 @@ import ownCloudSDK
 import ownCloudAppShared
 
 @available(iOS 13.0, *)
-public class CreateFolderIntentHandler: NSObject, CreateFolderIntentHandling {
+typealias CreateFolderCompletionHandler = (CreateFolderIntentResponse) -> Void
+
+@available(iOS 13.0, *)
+public class CreateFolderIntentHandler: NSObject, CreateFolderIntentHandling, OCCoreDelegate {
+	weak var core : OCCore?
+	var completionHandler: CreateFolderCompletionHandler?
+
+	func complete(with response: CreateFolderIntentResponse) {
```

Comment:
This function is duplicated in some classes you changed (except of the parameter type). Does it makes sense to put this code into the superclass with the generic type INIntentResponse as parameter?

Reply:
Please see https://github.com/owncloud/ios-app/pull/930#discussion_r603624730.